bedrock/firefox/urls.py

```diff
@@ -208,6 +207,8 @@
         "firefox/browsers/mobile/compare.html",
         ftl_files=["firefox/browsers/mobile/compare", "firefox/browsers/compare/shared"],
     ),
+    page("firefox/browsers/mobile/get-app/", "firefox/browsers/mobile/get-app.html", ftl_files=["firefox/mobile"]),
+
```

Comment:
Oh, it looks like our linter also wants this blank line to be removed. If you do that then the CI check here in the pull request should be happy again.